repository/zookeeper/README.md

```diff
@@ -0,0 +1,18 @@
+# Zookeeper Operator
+
+The KUDO Zookeeper operator creates, configures and manages [Apache Zookeeper](https://zookeeper.apache.org/) clusters running on Kubernetes.
+
+## Getting started
+
+The latest stable version of Zookeeper operator is `0.2.0`
+
+For the latest master branch you can check  [docs](./docs/latest) docs 
```

Comment:
we removed this line from Kafka docs also, so let's remove from here also and only give the latest stable version docs